script.js

```diff
@@ -243,8 +246,37 @@ function formLowerCaseValidate(submitForm) {
   }
 }
 
+function storageEmailData() {
```

Comment:
* Kindly fix this error which shows on the console on page load 
![image](https://user-images.githubusercontent.com/52497246/127598942-7ee53fc0-3954-47d4-9895-e482e8ca507d.png)

Reply:
Thank you Davidosky007 for your review :smile:, you are right, I wrote the code to save data in local storage when submit event occurs...

In order to improve the code next actions were taken:

1. Change functionalities so the value it's saved in local storage without clicking the submit button.
2. When the user changes the content of any input field, the data is saved to the local storage instantly.
3. The console error it's fixed.
![image](https://user-images.githubusercontent.com/82678907/127601739-8b99d9af-bfec-48b1-9895-794889c9fedb.png)

All the required changes are updated.